numpy/lib/type_check.py

```diff
@@ -149,9 +154,10 @@ def imag(val):
 
     Returns
     -------
-    out : ndarray
-        Output array. If `val` is real, the type of `val` is used for the
-        output.  If `val` has complex elements, the returned type is float.
+    out : ndarray or scalar
+        The imaginary component of a complex argument. If `val` is real,
+        the type of `val` is used for the output.  If `val` has complex
+        elements, the returned type is float.
```

Comment:
Not really `float`: `complex128` -> `float64`, `complex32` -> `float32`. But that was here before, s out of scope for this PR, I think